modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,954 @@
+    'pipe_db_server'            => '',                                                          # host for pipe db
+    'databases_server'          => '',                                                          # host for general output dbs
+    'dna_db_server'             => '',                                                          # host for dna db
+    'pipe_db_port'              => '',                                                          # port for pipeline host
+    'databases_port'            => '',                                                          # port for general output db host
+    'dna_db_port'               => '',                                                          # port for dna db host
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'species_name'              => '',                                                          # e.g. mus_musculus
+    'production_name'           => '',                                                          # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'uniprot_set'               => '',                                                          # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '',                                                          # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',                                                         # This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_projection'           => '0',                                                         # Will skip projection process if 1
+    'skip_rnaseq'               => '0',                                                         # Will skip rnaseq analyses if 1
+    'skip_cleaning'             => '0',                                                         # Will skip the cleaning phase, will keep more genes/transcripts but some lower quality models may be kept
+
+########################
+# Pipe and ref db info
+########################
+
+    'cdna_db_server' => $self->o('databases_server'),
+    'cdna_db_port'   => $self->o('databases_port'),
+
+    'genblast_db_server' => $self->o('databases_server'),
+    'genblast_db_port'   => $self->o('databases_port'),
+
+    'genblast_rnaseq_support_db_server' => $self->o('databases_server'),
+    'genblast_rnaseq_support_db_port'   => $self->o('databases_port'),
+
+    'ig_tr_db_server' => $self->o('databases_server'),
+    'ig_tr_db_port'   => $self->o('databases_port'),
+
+    'genewise_db_server' => $self->o('databases_server'),
+    'genewise_db_port'   => $self->o('databases_port'),
```

Comment:
Could this be changed to best_targeted_db instead of genewise_db? This is to avoid confusion